### ISSUE TITLE
Documentation searchability - 404s, outdated repos

### ISSUE THREAD

**Issue by CaelumF** [korlibs/korge#1996]
Hello,

\<appreciation message irrelevant to progress>
Love the work y'all are doing here! I'm making a jetpack compose multiplatform project targetting desktop and web via js/wasm, and for so many purposes, korlibs have saved me a LOT of pain. I understand they are originally made for the purpose of game development, which has its own unique set of challenges, and these libraries are super high quality production "serious application" grade and they deserve more recognition for just how useful they are outside of game development, too. And I say "y'all", because all OSS is a group effort, but it seems to be mostly just you @soywiz, which is super impressive
\<appreciation message irrelevant to progress>

It seems a lot of utility can be created by a few documentation discoverability improvements.

Ideal flow:
1. as a developer, I am using klock / korlib time, and I am using DateTimeTz. I want to represent a timezoned timerange, so I search to find if there is something for that purpose, similar to DateTimeSpan, but with timezones in mind
2. I do a google search for "korlibs DateTimeSpanTz"
3. if relevant github issues exist, they are shown, otherwise I go to nearest docs or github page

Current flow:
1. omitted for brevity
2. I do a google search for "korlibs DateTimeSpanTz"
3. https://github.com/korlibs/korlibs  shows as 2nd result, which is outdated? though not explicitly labeled as outdated. There are few issues, some may think this is it and just nobody has spoken much.
4. it contains submodules, one for klock which links to https://github.com/korlibs-archive/klock/tree/be84957186ad3805f4ce7abef034f02d1a6224e0
5. this repo says it has migrated to https://github.com/korlibs-archive/korge-next
6. korge-next says it has migrated to https://github.com/korlibs/korge
7. korge repo is up to date, and has a link to the docs at https://docs.korge.org/getting-started/, which are really nice btw!

Some adjacent searches link to things like https://docs.korge.org/klock/dates/ which 404 and some the old klock repo links to https://github.com/korlibs/korge/tree/main/klock which is also a 404

Searching korge repo issues is fine once I end up there. I do like separate repo pages for discoverability, but I understand that comes with quite an overhead too so I understand them all being one.

Probably all that should be done is for archived repos to directly link to https://github.com/korlibs/korge, and possibly for https://github.com/korlibs/korlibs to be archived and a link added too?

Some SEO on the docs would be awesome but I don't know much about SEO and potentially that is quite a hassle, maybe someone who does know about SEO can chime in though.

**Comment by soywiz** [comment]
Yeah, it is a bit problematic. The korlibs 4 are in a separate repo.
I believe this issue could be split into separate issues for specific actions. So feel free to do so.

Also, as a suggestion. Did you know that ChatGPT knows about klock? Might be useful:
https://chat.openai.com/share/985727d4-1ced-444e-9436-cf88b7023f35
